precommit/core.d/patchfiles.sh

```diff
@@ -327,9 +327,10 @@ function patchcmd_apply
 function patchfile_apply_driver
 {
   declare patchfile=$1
+  declare gpg=$2
 
   if declare -f ${PATCH_METHOD}_apply >/dev/null; then
-    "${PATCH_METHOD}_apply" "${patchfile}"
+    "${PATCH_METHOD}_apply" "${patchfile}" "${gpg}"
```

Comment:
Yes. Adding `-S` flag to the `gitapply_apply` will not cause anything. It will basically not be consumed, since `$2` will never have been used.
Also, `-S` is not a valid flag at all for git-apply. It will just fail out.